src/backend/iap/index.ts

```diff
@@ -116,7 +116,7 @@ export interface IapProduct {
 }
```

Comment:
For the `owner`, we could update the description to say "Object that represents the app creator ("seller")."

Reply:
Fixed.

Also replaced the description of *ownerUserId* with "Koji username of the seller."